src/app/static/src/app/featureSwitches.ts

```diff
@@ -1,7 +1,9 @@
 const urlParams = new URLSearchParams(window.location.search);
 const shareProject = !!urlParams.get('shareProject');
 const promoteProject = !urlParams.get('promoteProject');
+const renameProject = !urlParams.get('renameProject');
```

Comment:
this switch is turned on at the moment - just need one more `!` here to turn it off